serverless/tensorflow/faster_rcnn_inception_v2_coco/nuclio/function-gpu.yaml

```diff
@@ -114,7 +114,8 @@ spec:
         - kind: RUN
           value: ln -s faster_rcnn_inception_v2_coco_2018_01_28 faster_rcnn
         - kind: RUN
-          value: pip install pillow pyyaml
+          value: pip install pillow<10 pyyaml
+          # faster_rcnn code used methods that are not supported on Pillow>10
```

Comment:
Wait... this function doesn't include any third-party code. So all Pillow-related code should be ours, in which case... can't we just fix it?

Reply:
Yes, you're right. I didn't read logs well. Pillow turned out to be used in our part, replaced depricated method `ANTIALIAS` to `LANCZOS` by [relesae notes](https://pillow.readthedocs.io/en/stable/releasenotes/10.0.0.html#constants).